docs/userGuide/tweakingThePageStructure.md

```diff
@@ -306,4 +306,63 @@ In this case `myHead.md` will override the `chapterLayout/head.md`.
 
 **The layout named `default` (if any) is automatically applied to every single page.** When you `init` a MarkBind site, MarkBind also generates an empty `default` layout.
 
+## Toggling alternative contents in a page
+
+You can use tags to selectively filter HTML elements when building a site.
+
+Tags are specified by the `tags` attribute, **and can be attached to any HTML element**.
+
+<div class="indented">
+
+{{ icon_example }} Attaching tags to elements:
+```html
+# Print 'Hello world'
+
+<p tags="language--java">System.out.println("Hello world");</p>
+<div tags="language--C#">Console.WriteLine("Hello world");</div>
+<span tags="language--python">print("Hello world")</span>
+```
+</div>
+
+You need to specify the tags to include in `site.json`, under the `tags` option:
+
+```
+{
+  ...
+  "tags": ["language--java"]
+}
+```
+
+During rendering, only elements that match tags specified in the `site.json` files will be rendered. All other tagged elements will be filtered out. In this case, only the element with the `language--java` tag will be rendered. This is helpful when creating multiple versions of a page without having to maintain separate copies.
```

Comment:
The first statement should probably come right after the statement `Tags are specified by the `tags` attribute...`, since at this portion of the text, it should be explaining the example, not giving additional comments about the feature.

The second statement should also be indented together with the example.